lib/castle/hourly_download.ex

```diff
@@ -29,7 +31,22 @@ defmodule Castle.HourlyDownload do
     length(rows)
   end
 
+  def set_podcast_totals! do
```

Comment:
These aren't quick.  Something like ~10s in prod.  But I guess that's fine, since it's a background task?